packages/ts/react-crud/src/autoform.tsx

```diff
@@ -391,7 +399,12 @@ export function AutoForm<M extends AbstractModel>({
     );
   }
 
-  const visibleProperties = visibleFields ? modelInfo.getProperties(visibleFields) : getDefaultProperties(modelInfo);
+  let visibleProperties = visibleFields ? modelInfo.getProperties(visibleFields) : getDefaultProperties(modelInfo);
+
+  // When using `hiddenFields`, remove fields to hide using their `name`
+  if (hiddenFields) {
+    visibleProperties = visibleProperties.filter(({ name }) => !(name && hiddenFields.includes(name)));
```

Comment:
```suggestion
    visibleProperties = visibleProperties.filter(({ name }) => !hiddenFields.includes(name));
```
`name` has type `string`.